src/pages/ProfilePage.tsx

```diff
@@ -658,10 +658,13 @@ const ProfileScoreCard = (userScore: UserScore) => {
               direction={{ xs: "column", sm: "row" }}
               spacing={{ sm: 1 }}
               alignItems={{ xs: "flex-start", sm: "center" }}
+              maxWidth="15vw"
             >
-              <Typography variant="h6">{songName}</Typography>
-              <Typography variant="body1" fontWeight="lighter">
-                by {artist}
+              <Typography variant="h6" noWrap={true}>
+                {songName}&nbsp;
+                <Box component="span" fontWeight="lighter" fontSize="1rem">
+                  by {artist}
+                </Box>
```

Comment:
Maybe in the long run we should consider truncating artist & song name separately